app/helpers/users_helper.rb

```diff
@@ -1,2 +1,20 @@
 module UsersHelper
+  def render_user_posts(user)
+    posts_html = ''
+    user.posts.each do |post|
+      posts_html += content_tag(:div, class: 'post') do
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=page%20to%20another-,Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.,-Hint%3A%20In), You used this method to render posts in user show view, please use the `recent_posts` method inside user model to render the most recent 3 posts inside `user show view`.